iiod/iiod.c

```diff
@@ -455,7 +465,11 @@ int main(int argc, char **argv)
 		}
 	}
 
-	ctx = iio_create_local_context();
+	if (arg)
+		ctx = iio_create_context_from_uri(arg);
```

Comment:
Initialize "arg" to `local:`, then you can unconditionally call iio_create_context_from_uri().

Reply:
done